classes/class-p4-campaign-importer.php

```diff
@@ -281,5 +282,26 @@ public function process_campaign_metas( $post_meta ) {
 
 			return $post_meta;
 		}
+
+		/**
+		 * Skip already existing postmeta data from import.
+		 *
+		 * @param array   $campaign_postmeta The to be imported post meta fields.
+		 * @param integer $post_id Post ID.
+		 *
+		 * @return array The cleaned post meta fields.
+		 */
+		public function skip_duplicate_postmeta_import( $campaign_postmeta, $post_id ) {
```

Comment:
You can make it just `$postmeta`, as this same function is called for the meta of the attachments in the import file too.